x-pack/test_utils/jest/config.js

```diff
@@ -20,6 +20,7 @@ export default {
     'server/**/*.js',
   ],
   moduleNameMapper: {
+    '^plugins/([^\/.]*)(.*)': '<rootDir>/src/legacy/core_plugins/$1/public$2',
     '^ui/(.*)': '<rootDir>**/public/$1',
     '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$': '<rootDir>/src/dev/jest/mocks/file_mock.js',
```

Comment:
I think this module map would never have worked, since it should refer to `<rootDir>/../src/dev/...` since the root dir of the x-pack tests is the x-pack folder itself? cc @spalger